activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -112,7 +112,7 @@ def indexes(table_name) # :nodoc:
             inddef = row[3]
             comment = row[4]
             valid = row[5]
-            columns = decode_string_array(row[6])
+            columns = decode_string_array(row[6]).map { |c| Utils.unquote_identifier(c.strip.gsub('""', '"')) }
```

Comment:
what do you think of moving the map inside `decode_string_array` or should this be used only in this context?

Reply:
it is possible foreign keys have the same problem, if yes, then probably a good idea. Will take a look